package.json

```diff
@@ -16,7 +16,10 @@
     "build-tests": "mkdir -p dist/ && browserify test/karma-qunit-shim.js test/*-test.js -u test/karma.conf.js -s videojs-dock -d -o dist/tests.js",
     "watch-tests": "mkdir -p dist/ && watchify test/karma-qunit-shim.js test/*-test.js -u test/karma.conf.js -s videojs-dock -v -d -o dist/tests.js",
     "test": "jshint src/**.js test/**.js index.js && npm run build-tests && karma start test/karma.conf.js --single-run",
-    "prepublish": "npm run build",
+    "predist": "npm run babel",
+    "dist": "mkdir -p dist/ && browserify -t browserify-shim . -s videojs-dock -o dist/bundle.js",
```

Comment:
The `dist` file needs to be browserified with `browserify-shim` as a transform. It's still configured below in this file.